iceoryx_hoofs/source/posix_wrapper/named_pipe.cpp

```diff
@@ -153,6 +158,8 @@ template <typename Prefix>
 IpcChannelName_t NamedPipe::convertName(const Prefix& p, const IpcChannelName_t& name) noexcept
 {
     return IpcChannelName_t(cxx::TruncateToCapacity,
+                            // name is checked for emptyness, so it's ok to get a first member
+                            // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
```

Comment:
This comment is wrong, the name is not checked in this method and could be empty